core/src/banking_stage/consume_worker.rs

```diff
@@ -376,6 +391,8 @@ struct ConsumeWorkerCountMetrics {
     retryable_transaction_count: AtomicUsize,
     retryable_expired_bank_count: AtomicUsize,
     cost_model_throttled_transactions_count: AtomicUsize,
+    scheduled_min_prioritization_fees: AtomicUsize,
+    scheduled_max_prioritization_fees: AtomicUsize,
```

Comment:
I think these are misleadingly named. These aren't scheduled, the min/max are coming from what we are processing which is not quite the same.
We can schedule txs, but if they took too long to process and the slot ends we may not process them.

Naming would be less confusing if we just dropped the prefix imo.